Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.Properties.cs

```diff
@@ -42,7 +42,7 @@ public partial class HeaderedTextBlock
             nameof(Text),
             typeof(string),
             typeof(HeaderedTextBlock),
-            new PropertyMetadata(null));
+            new PropertyMetadata(null, (d, e) => { ((HeaderedTextBlock)d).UpdateText(); }));
```

Comment:
Since all this method does is call UpdateVisibility, why not just call UpdateVisibility from here instead? I guess the same applies to the Header method call too.